### DISCUSSION TITLE
[220440]鬼泣DMC/DmC Devil May Cry

### DISCUSSION THREAD

**Discussion by dongxuewanqing** [pjy612/SteamManifestCache#742] [Share]
### Steam 游戏ID

220440

### 密钥列表

```hash
220441;aed2b190f9b8ea15e6f1e22aed2d339f162f17b8673f06124dd11e1ea45b11be
```

### 清单附件

[220440.zip](https://github.com/user-attachments/files/16321496/220440.zip)

### 备注

237372140@qq.com

**Comment by pjy612** [comment]
https://steamdb.info/app/220440/depots/?branch=public 
全DLC的话 密钥可不止这么一条吧？

**Reply by dongxuewanqing** [reply]
改了

**Reply by dongxuewanqing** [reply]
我DLC不全

**Comment by barryhamsy** [comment]
hey, thanks for sharing, i want to know in the .lua, where do you get this lines?
setManifestid(220441,"6406147301489057498",8992495131)
